Dockerfile.QA

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Accidentally bumped copyright date without any changes in other [PR](https://github.com/triton-inference-server/server/pull/5119/files#diff-0c3eb9bfef58682e83232ffa9140d7cab2bbcc40488da3dc0f6dabb474296f2f), so fixing here